lib/torrent.js

```diff
@@ -1545,10 +1547,7 @@ class Torrent extends EventEmitter {
         if (hash === self._hashes[index]) {
           if (!self.pieces[index]) return
           self._debug('piece verified %s', index)
-
-          self.pieces[index] = null
-          self._reservations[index] = null
-          self.bitfield.set(index, true)
+          self._markVerified(index)
```

Comment:
Why this code removed?

Reply:
code reuse.  those same calls are in _markVerified